client/src/app/components/pull-request-table/pull-request-table.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component, inject, signal } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TableModule } from 'primeng/table';
+import { AvatarModule } from 'primeng/avatar';
+import { TagModule } from 'primeng/tag';
+import { injectQuery } from '@tanstack/angular-query-experimental';
+import { tap } from 'rxjs';
+import { IconsModule } from 'icons.module';
+import { PullRequestControllerService, PullRequestInfoDTO } from '@app/core/modules/openapi';
+
+
+@Component({
+  selector: 'app-pull-request-table',
+  imports: [CommonModule, TableModule, AvatarModule, TagModule, IconsModule],
+  templateUrl: './pull-request-table.component.html',
+  styles: [`
+    :host ::ng-deep {
```

Comment:
I am not an expert on this but is it ok to use `ng-deep` in component or in general?